src/Dialogs/BulkRenamer/RenamerDialog.vala

```diff
@@ -185,7 +192,7 @@ public class Files.RenamerDialog : Gtk.Dialog {
         var content_box = get_content_area ();
         content_box.pack_start (controls_grid);
         content_box.pack_start (list_grid);
-        content_box.margin = 12;
+        content_box.margin = 10;
```

Comment:
this keeps things in line since dialog comes with a border of 2px we can't get rid of for some reason